javascript-carousel/main.js

```diff
@@ -0,0 +1,63 @@
+var $leftArrow = document.querySelector('.fa-angle-left');
+var $rightArrow = document.querySelector('.fa-angle-right');
+var $pokemonImage = document.querySelector('.pokemon-image');
+var $pokemonArray = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var $circlesRow = document.querySelector('.all-circles');
+var $allCircles = document.querySelectorAll('.circle');
+var $carousel = setInterval(moveRight, 3000);
+
+function moveLeft(event) {
+  for (var i = $pokemonArray.length - 1; i >= 0; i--) {
+    if ($pokemonImage.getAttribute('src') === 'images/001.png') {
+      $pokemonImage.setAttribute('src', $pokemonArray[i]);
+      $allCircles[i].className = 'fas fa-circle circle';
+      $allCircles[0].className = 'far fa-circle circle';
+      resetInterval();
+      return;
```

Comment:
What if you ever needed to reorganize the order of your images? I wouldn't organize your logic around the images shown being in exactly that order. You should make a variable to keep track of the index of the current image in your array.